tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,95 @@
+    def test_memorial_day(self):
+        self.assertHolidayName("Хатыра гүни (Memorial Day)", (f"{year}-01-12" for year in range(1992, 2050)))
+
+    def test_defenders_day(self):
+        name = "Ватанмухадызларың гүни"
+        for year in range(2009, 2050):
+            self.assertHolidayName(name, f"{year}-01-27")
+        for year in range(1992, 2009):
+            self.assertNoHolidayName(name, year)
+
+    def test_womens_day(self):
+        self.assertHolidayName("Халықаралық әйелдер күні", (f"{year}-03-08" for year in range(1992, 2050)))
```

Ruff (0.8.2)

Line too long (107 > 99)

(E501)